### PR TITLE
Feature/qa debug

### PR DESCRIPTION
**Pull request by activemq**
- [x] identify cause of issue test fails
- [x] Redis ruby access blocked by firewall
- [x] Redis ruby code refactored to be explicit about why value not found
- [x] Redis inaccessible - fail entire test suite run and exit early

### CODE REVIEW

File: test/admin-hpa/features/support/env.rb [hunk 1]
```diff
@@ -61,7 +61,6 @@
   config.exact = true
   config.ignore_hidden_elements = false
   config.visible_text_only = true
-  config.default_max_wait_time = 7
```
Comment (GuyHarwood) [inline test/admin-hpa/features/support/env.rb:64]: when its not clear what unit the config item requires we create a variable that describes it, and assign that to the property.
Like this...
```
sevenSeconds = 7
config.default_max_wait_time = sevenSeconds
```

File: test/pupil-hpa/features/support/redis_helper.rb [hunk 2]
```diff
@@ -2,16 +2,17 @@ class RedisHelper
 
   def self.wait_for_prepared_check(school_password,pin)
     p school_password, pin
-    found = false
     begin
       retries ||= 0
       sleep 5
```
Comment (GuyHarwood) [inline test/pupil-hpa/features/support/redis_helper.rb:7]: same as above, use a named variable with the unit in its name

File: test/admin-hpa/features/support/env.rb (hunk 1 above)
Comment (activemq) [reply test/admin-hpa/features/support/env.rb:64]: Will do going forward, this config was removed as part of this commit

File: test/pupil-hpa/features/support/redis_helper.rb (hunk 2 above)
Comment (activemq) [reply test/pupil-hpa/features/support/redis_helper.rb:7]: Will do as above

**Review by GuyHarwood — APPROVED** [review]
LGTM

**Pull request merged**